### ISSUE TITLE
Explicitly enable autoescaping by default

### ISSUE THREAD

**Issue by gcmurphy** [aio-libs/aiohttp-jinja2#177]
The jinja2 engine does not currently enable autoescaping by default. From http://jinja.pocoo.org/docs/2.10/api/ 

> In future versions of Jinja2 we might enable autoescaping by default for security reasons. As such you are encouraged to explicitly configure autoescaping now instead of relying on the default.

Although the option to configure this is exposed via aiohttp_jinja2.setup, it might be nice to include it as a default option or explicitly call out the need to enable it in examples and in the documentation.  

XXS demo: 

```html
<!-- index.j2 -->
<html>
    <p>No autoescape by default?</p>
    <p>{{ thing }}</p>
</html>
```
```python
# main.py
from aiohttp import web
import aiohttp_jinja2
import jinja2

@aiohttp_jinja2.template('index.j2')
async def index(request):
    return { 'thing': '<script>alert(1)</script>' }

app = web.Application()
aiohttp_jinja2.setup(app, loader=jinja2.FileSystemLoader('.'))

app.router.add_get('/', index)
web.run_app(app)
```

**Comment by asvetlov** [comment]
Make sense.
Would you provide a pull request?

**Comment by asvetlov** [comment]
Fixed by #178
